lib/actions/google/drive/sheets/google_sheets.js

```diff
@@ -194,13 +194,13 @@ class GoogleSheetsAction extends google_drive_1.GoogleDriveAction {
                 throw "No spreadsheet ID";
             }
             const spreadsheetId = files.data.files[0].id;
-            const sheets = yield this.retriableSpreadsheetGet(spreadsheetId, sheet, 0, request.webhookId).catch((e) => {
+            const sheets = yield this.retriableSpreadsheetGet(spreadsheetId, sheet, 0, request.webhookId)
+                .catch((e) => {
                 this.sanitizeGaxiosError(e);
                 winston.debug(`Error retrieving spreadsheet. Error ${e.toString()}`, { webhookId: request.webhookId });
                 throw e;
             });
-            if (!sheets.data.sheets ||
-                sheets.data.sheets[0].properties === undefined ||
+            if (!sheets.data.sheets || sheets.data.sheets[0].properties === undefined ||
                 sheets.data.sheets[0].properties.gridProperties === undefined) {
                 throw "Now sheet data available";
```

Comment:
What is this error message trying to say?

Reply:
This is generated JS and I wasn't changing it. This is just aftereffects of fixing the formatting